cmd/kots/cli/admin-console-upgrade.go

```diff
@@ -51,9 +52,11 @@ func AdminConsoleUpgradeCmd() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().Bool("force-upgrade-kurl", false, "set to force upgrade even if this is a kurl cluster")
```

Comment:
If this is enabled, let's print an message on the screen that we are proceeding with something that's not recommended